### DISCUSSION TITLE
Watch individual GitHub issue labels and receive notifications

### DISCUSSION THREAD

**Discussion by davidfstr** [community/community#16645] [Projects and Issues]
I am personally interested in watching a [specific label](https://github.com/python/cpython/labels/expert-typing) on a [specific repository](https://github.com/python/cpython) such that whenever:

(1) a new issue is created with that label or 
(2) that label is added to an existing issue

then I get a notification.

It is already possible for me to watch _all_ issues in a particular repository, but that is more than I want to get notified about.

Known workarounds:

* [github-label-notify](https://github.com/bcongdon/github-label-notify) - A custom IFTTT template recipe that will watch for when specific labels are added to issues within a particular repository.

* [GitHub Action - Issue Label Notifications](https://github.com/marketplace/actions/issue-label-notifier) - Per-repository GitHub Action that watches for the add of a specific label and notifies specific people. Requires write access to the watched repository.

**Comment by Okabe-Junya** [comment]
I also hope for the implementation of this feature. I would like to use this feature in the following use cases:

- Receive notifications for issues labeled `good first issue` in OSS projects that I am interested in contributing to
- Receive notifications for issues labeled `lang/ja` in projects where I am contributing to Japanese localizations

From my research, it seems that this feature is available as an early access preview in some projects. (I'm not sure if it's still available now...)

- https://discourse.llvm.org/t/label-based-notification/60517

I am looking forward to this feature being available to all users!!

**Comment by qiweiii** [comment]
Hi I also want to get notified for labels and other custom events, then I build this extension recently: https://github.com/qiweiii/github-custom-notifier. Basically I use gtihub API to poll data every 2 minutes, with [issue events](https://docs.github.com/en/rest/issues/events?apiVersion=2022-11-28) and [issue comment](https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#list-issue-comments-for-a-repository) api, I can get lots of custom events that's not available on GitHub notifications, watches and emails

**Reply by Julian-Dumitrascu** [reply]
You welcome contributions. I can help write instructions that more people would understand.
People also like to understand the benefits and costs of a service. 
It's not easy for me to estimate the costs of using this browser extension. I think I don't understand your instructions. They seem incomplete. 
There is a difference between the author understanding what they are describing and users understanding those 2 things. What is your goal? Since you ask for money, would you like to persuade more people to pay?

**Comment by ZubairImtiaz3** [comment]
Hey everyone, I built a web app called [GitIssuefy](https://git-issuefy.vercel.app/), which allows you to track issues in GitHub repositories based on labels.

Notifications are sent directly to your Discord via DM, and you can track multiple label combos together or individually.

With GitIssuefy, you can get notified:
- When a new issue is created with a specific label.
- When a label is added to an existing issue.
- When a multiple-label combo you're watching is completed on an issue.

Feel free to check it out! The project is open-source and free, so if you're interested, you can contribute. 
Here's the link to the repository: [GitIssuefy GitHub](https://github.com/ZubairImtiaz3/GitIssuefy).

**Reply by Julian-Dumitrascu** [reply]
One can achieve this with one computer program instead of these two.

**Comment by Dvd-Znf** [comment]
This would also be really useful for Pull Requests,
A lot of automated bots are made for simply @ notifying people
